geode-core/src/integrationTest/java/org/apache/geode/distributed/internal/tcpserver/TcpServerJUnitTest.java

```diff
@@ -81,21 +81,20 @@ private void start(TcpHandler handler) throws IOException {
   }
 
   @Test
-  public void test() throws Exception {
-    EchoHandler handler = new EchoHandler();
+  public void testClientGetInfo() throws Exception {
```

Comment:
Eventually we probably want to move this test (Along with InfoRequest/Response) out of the tcp server package.